miner/README.org

```diff
@@ -60,7 +60,7 @@ Pair before beginning.
 ** Obtaining a Key Pair
 
 The easiest way to obtain a Key Pair is via the [[https://pact.kadena.io/][Pact Web]] application. Keys can
-be generated by the /Wallet/ widget in the bottom-right. Once generated, take
+be generated by the /Wallet/ widget in the bottom right. Once generated, take
```

Comment:
My intuition says that the hyphen is correct, but the internet seems unable to agree. :thinking:

Reply:
Space looks better to my eye.

Reply:
From my understanding, hyphens are needed when one or two more words (meant to modify) are placed before a noun. 

**Ex**: Keys can be generated by the Wallet widget in the bottom right vs. The keys are generated by the Wallet widget on the bottom-right corner.